docs/source/format/CanonicalExtensions.rst

```diff
@@ -72,4 +72,28 @@ same rules as laid out above, and provide backwards compatibility guarantees.
 Official List
 =============
 
-No canonical extension types have been standardized yet.
+Fixed size tensor
+=================
+
+* Extension name: `arrow.fixed_size_tensor`.
+
+* The storage type of the extension: ``List``.
```

Comment:
I think we'd want [FixedSizeList](https://github.com/apache/arrow/blob/master/format/Schema.fbs#L99) for fixed size tensor.
```suggestion
* The storage type of the extension: ``FixedSizeList``.
```